src/k8s/K8sWorkerController.cpp

```diff
@@ -182,6 +279,9 @@ std::string K8sWorkerController::getMasterIp() const { return this->masterIp; }
 
 int K8sWorkerController::getNumberOfWorkers() const { return numberOfWorkers; }
 
+/*
+ * @deprecated
```

Comment:
Why do we mark deprecated here? Please put details in the code comment. Also have we followed the standard in marking code segment as deprecated here?

Reply:
We can remove that function. Actually, it is not used currently. I'll remove that function

Reply:
Ok